setup/setup.bash

```diff
@@ -307,7 +307,12 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
        fi
    else
        echo "Installing LibMagic with HomeBrew"
-       /usr/local/bin/brew install libmagic
+       OUTPUT="$(/usr/local/bin/brew install libmagic 2>&1 > /dev/null)"
```

Comment:
I think this is wrong. You’re sending stderr to stdout with “2>&1” and then you send stdout to dev null. So what’s OUTPUT going to be? The return code? Poor choice of var name then ;-) I think you should remove the part where you send stdout to dev null: “> /dev/null”. Just remove that section of the command.